.vsts-ci.yml

```diff
@@ -65,13 +86,22 @@ jobs:
       echo "##vso[task.prependpath]/usr/lib64/openmpi/bin"
       echo "##vso[task.prependpath]$CONDA/bin"
     displayName: 'Set variables'
+  - script: |
+      git clean -d -f -x
```

Comment:
@shiyu1994 we are only running one agent per machine in the linux pool for Azure DevOps, right?

I saw that it's possible to run multiple ([Azure Pipelines Agents docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=yaml%2Cbrowser#can-i-install-multiple-self-hosted-agents-on-the-same-machine)).

If we were running multiple agents on the same virtual machine, then destructive actions like `git clean` like this might cause conflicts between different jobs.